osfv_cli/src/osfv/libs/rte.py

```diff
@@ -223,10 +223,14 @@ def pwr_ctrl_after_flash(self, programmer):
             time.sleep(2)
 
     def flash_cmd(self, args, read_file=None, write_file=None):
-        self.pwr_ctrl_before_flash(
-            self.dut_data["programmer"]["name"],
-            self.dut_data["pwr_ctrl"]["flashing_power_state"],
-        )
+        try:
+            self.pwr_ctrl_before_flash(
+                self.dut_data["programmer"]["name"],
+                self.dut_data["pwr_ctrl"]["flashing_power_state"],
+            )
+        except requests.exceptions.ConnectionError as e:
+            print(f"Failed to change power state while flashing")
```

Comment:
```suggestion
            print("Failed to change power state while flashing")
```

No formatting of the string so `f` is unneeded here. Unless the formatting was supposed to be here?

E.g.

```
            print(f"Failed to change power state while flashing: {e}")
```